src/prefect_server/services/towel/zombie_killer.py

```diff
@@ -180,7 +180,10 @@ async def reap_zombie_task_runs(
 
                 # mark failed
                 else:
-                    message = "No heartbeat detected from the remote task; marking the run as failed."
+                    message = (
+                        "No heartbeat detected from the remote task; marking the "
+                        f"task run {tr.task.id} ({tr.task.name}) as failed."
```

Comment:
This logs the task ID and name, not the task run ID and name; I think you meant:
```suggestion
                        f"task run {tr.id} ({tr.name}) as failed."
```
in which case the expanded query above isn't necessary

Reply:
@cicdw Thanks! I added name to query because didn't have. I runned tests locally is normal task run name be null?

Reply:
Yes that is normal - task runs only have names when a user explicitly provides them.

Reply:
Great, thanks!

Reply:
Thanks @cicdw, I lookup again the issue and noticed that name is indeed task name not task run name, should i change the PR acordelly?